accord-core/src/test/java/accord/impl/list/ListRequest.java

```diff
@@ -110,49 +122,84 @@ public void accept(Result success, Throwable fail)
                     return;
                 }
 
-                node.reply(client, replyContext, new ListResult(client, ((Packet)replyContext).requestId, txnId, null, null, new int[0][], null));
-                ((Cluster)node.scheduler()).onDone(() -> {
-                    node.commandStores()
-                        .select(homeKey)
-                        .execute(() -> CheckOnResult.checkOnResult(node, txnId, homeKey, (s, f) -> {
-                            if (f != null)
-                            {
-                                node.reply(client, replyContext, new ListResult(client, ((Packet) replyContext).requestId, txnId, null, null, f instanceof Truncated ? new int[2][] : new int[3][], null));
-                                return;
-                            }
-                            switch (s)
-                            {
-                                case Truncated:
-                                    node.reply(client, replyContext, new ListResult(client, ((Packet) replyContext).requestId, txnId, null, null, new int[2][], null));
-                                    break;
-                                case Invalidated:
-                                    node.reply(client, replyContext, new ListResult(client, ((Packet) replyContext).requestId, txnId, null, null, null, null));
-                                    break;
-                                case Lost:
-                                    node.reply(client, replyContext, new ListResult(client, ((Packet) replyContext).requestId, txnId, null, null, new int[1][], null));
-                                    break;
-                                case Other:
-                                    // currently caught elsewhere in response tracking, but might help to throw an exception here
-                            }
-                        }));
-                });
+                node.reply(client, replyContext, ListResult.heartBeat(client, ((Packet)replyContext).requestId, txnId));
+                ((Cluster) node.scheduler()).onDone(() -> checkOnResult(homeKey, txnId, 0, null));
+            }
+            else if (fail instanceof SimulatedFault)
+            {
+                ((NodeSink) node.messageSink()).debugClient(id, fail, NodeSink.ClientAction.FAILURE);
+                node.reply(client, replyContext, ListResult.heartBeat(client, ((Packet)replyContext).requestId, id));
+                ((Cluster) node.scheduler()).onDone(() -> checkOnResult(null, id, 0, null));
+            }
+            else
+            {
+                ((NodeSink) node.messageSink()).debugClient(id, fail, NodeSink.ClientAction.FAILURE);
+                node.agent().onUncaughtException(fail);
+            }
+        }
+
+        private void checkOnResult(@Nullable RoutingKey homeKey, TxnId txnId, int attempt, Throwable t) {
+            if (attempt == 42)
```

Comment:
I think this was left in erroneously

Reply:
I think I used to have 3 or something, `42` was to see if the issue was ephemeral or not (the issue was not); I can lower the retry limit